source/dom/platform.js

```diff
@@ -45,6 +45,8 @@ enyo.platform = {
 		{platform: "androidChrome", regex: /Android .* Chrome\/(\d+)[.\d]+/},
 		// Android 2 - 4
 		{platform: "android", regex: /Android (\d+)/},
+		// Some android phones, e.g. ZTE and Lenovo
+		{platform: "android", regex: /Android\/(\d+)/},
```

Comment:
I think there are a fair number of devices with this user agent string pattern, where it would make sense to combine these: http://www.botsvsbrowsers.com/listings.asp?search=android%2F4.